packages/components/src/chart/index.js

```diff
@@ -228,14 +249,14 @@ class Chart extends Component {
 	}
 
 	getLegendPosition() {
-		const { legendPosition, mode, isViewportWide } = this.props;
+		const { legendPosition, mode, isViewportWide, isViewportMedium } = this.props;
 		if ( legendPosition ) {
 			return legendPosition;
 		}
 		if ( isViewportWide && mode === 'time-comparison' ) {
 			return 'top';
 		}
-		if ( isViewportWide && mode === 'item-comparison' ) {
+		if ( isViewportMedium && mode === 'item-comparison' ) {
```

Comment:
This change is making the chart to look very small in narrow viewports:

![image](https://user-images.githubusercontent.com/3616980/51754837-98719880-20bd-11e9-9b30-d822b475b2e8.png)

AFAIK, the goal was to switch to a single column layout (the legend appearing below the chart) in viewports smaller than 1280px. Maybe @LevinMedia can confirm.

Reply:
Affirmative - lets switch to single column view <1280 thanks for the catch @Aljullu !

Reply:
Done in https://github.com/woocommerce/wc-admin/pull/1391/commits/275356e15585f88dc9f21d0a676dc6e111633c0d.